docs/js/main.js

```diff
@@ -893,15 +898,6 @@ onload = function() {
             }
         }
 
-        // resetting the type for starbattle composite mode

```

Comment:
Is there a reason for removing this?
I am unable to recollect the exact reason why I needed this but something around improving the composite mode behavior especially on mobiles/ipads.